javascript/package.json

```diff
@@ -29,6 +31,7 @@
     "@gomomento/generated-types": "0.20.0",
     "@gomomento/sdk": "0.14.3",
     "@grpc/grpc-js": "1.5.10",
-    "hdr-histogram-js": "3.0.0"
+    "hdr-histogram-js": "3.0.0",
+    "node-fetch": "^2.6.7"
```

Comment:
We need to specifically use v2 here because the newer v3 doesn't support CommonJS modules with typescript.